Make dataset the highest directory level in ia39 (#4)
AusClimateService/code-roadmap: xv83.md

> The `ia39` project could be (it is currently unused) a place to share data among researchers working on the ACS project.

Reviewer: "researchers working on the ACS project" -> "researchers working with ACS-related material"

This allows us to capture community use of pre-release ACS-related data, for example model validation and verification that is required prior to publication regardless of whether that process is part of ACS (for example with ACCESS CMIP6, field specialists required access to the 'authoritative' data for verification before it could be published as part of NESP, but the specialists were not necessarily NESP-affiliated researchers, I'm trying to capture that).

We also want to capture that ACS may provide a service to the Australian climate community through management of replica datasets that are not otherwise available at NCI but may be used by, say, Clex researchers as well as ACS researchers.

Reviewer 2: Agreed

> The `ia39` project could be (it is currently unused) a place to share data among researchers working on the ACS project.

Author: ```suggestion
The `ia39` project could be (it is currently unused) a place to share data among researchers working with ACS-related material.
```

> 

Reviewer: Yes, I think a README with most of that information would definitely be required. In particular it's really important with this type of data that caveats and limitations be very clearly indicated - for example it's likely that "authoritative" data is (probably?) not appropriate for publishing on, as it is "pre-QC" - data on which publications are based should be that in `pd00` or elsewhere published formally.

> A README file explaining when, how and what was downloaded from the authoritative source,

Author: ```suggestion
A README file explaining when, how and what was downloaded from the authoritative source,
the maintenance plan (i.e. is this replica static or is it being regularly updated and how),
```

> with appropriate links to dataset documentation. 

Reviewer: Also maintenance plan - is this replica static or is it being regularly updated? If so, by whom and how? (e.g. manual download quarterly, cron-like script monthly, etc).

> with appropriate links to dataset documentation. 

Author: ```suggestion
and links to dataset documentation. 
```
